WORKSPACE

```diff
@@ -0,0 +1,74 @@
+# The WORKSPACE file tells Bazel that this directory is a "workspace", which is like a project root.
+# The content of this file specifies all the external dependencies Bazel needs to perform a build.
+
+####################################
+# ESModule imports (and TypeScript imports) can be absolute starting with the workspace name.
+# The name of the workspace should match the npm package where we publish, so that these
+# imports also make sense when referencing the published package.
+workspace(name = "ngx_tabs_libdemo")
+
+
+####################################
+# The Bazel buildtools repo contains tools like the BUILD file formatter, buildifier
+http_archive(
+    name = "com_github_bazelbuild_buildtools",
+    # Note, this commit matches the version of buildifier in angular/ngcontainer
+    url = "https://github.com/bazelbuild/buildtools/archive/b3b620e8bcff18ed3378cd3f35ebeb7016d71f71.zip",
```

Comment:
Guess this is a temporary include and will later point to some release like the others further below, right?

Reply:
it could point to a release now, but like the comment says it's trying to match the version in the docker container. It would be super-annoying if the local formatting doesn't satisfy the CI lint check.

soon we'll have bazel remote caching working well enough that we don't need a binary in the docker image, we could fetch it from the cache.